geode-for-redis/src/main/java/org/apache/geode/redis/internal/statistics/RedisStats.java

```diff
@@ -144,24 +144,15 @@ public long getKeyspaceMisses() {
     return keyspaceMisses.get();
   }
 
-  public long startPassiveExpirationCheck() {
-    return getCurrentTimeNanos();
-  }
-
   public void endPassiveExpirationCheck(long start, long expireCount) {
     geodeRedisStats.endPassiveExpirationCheck(start, expireCount);
   }
 
-  public long startExpiration() {
```

Comment:
I would be in favor of keeping these "bookend" start methods. I know that is this particular case all that start does is return getCurrentTimeNanos. But I think it is worth preserving the start/end pattern. And it would make it simple in the future to add other starts that are changed on start (like an in progress stat). I know these particular start methods can be unwrapped but then the abstraction is different and it is not as obvious in the caller how the start and end calls correspond.

Reply:
Okay, I've put these back as they were.